src/utils/metrics.py

```diff
@@ -172,3 +228,21 @@ def log(self, stats: VllmStats) -> None:
             self.metrics.histogram_time_per_output_token,
             stats.time_per_output_tokens_iter,
         )
+        # Request stats
```

Comment:
I would suggest to refactor this as follows:
```python
triton_metrics = [ self.metrics.histogram_time_to_first_token, ... ]
vllm_stats = [ stats.time_to_first_tokens_iter, ... ]

for tuple in zip(triton_metrics, vllm_stats):
    self._log_histogram(tuple)
```
This way it is more extendable in my mind. Triton metrics are all defined in one place, as well as vLLM's stats. 
Potentially, maybe there's a way to extract all Triton metrics and vLLM stats without explicitly composing lists. I would recommend checking

Reply:
More on `zip`: https://realpython.com/python-zip-function/

Reply:
Addressed